content/docs/learning-guide/tutorials/environments/generate-terrain.md

```diff
@@ -0,0 +1,303 @@
+
+3. In the Camera entity, in the **Camera** component, choose **Be this camera** to view the level through the camera entity.
+
+4. In the viewport camera settings in the upper-right of the viewport {{< icon "camera.svg" >}} set the **Camera Speed** to an appropriate value for the size of the terrain. `100.0` is a good reference point for large terrains.
+
+These suggestions are meant to help you view the entire terrain while you build it in O3DE Editor. In Launcher applications, these settings should be set to reasonable values for performance considerations.
+{{< /important >}}
+
+## Terrain setup
+
+Generating terrain requires multiple entities that reference each other. You can set up the entities and references manually, or you can use [Landscape Canvas](/docs/user-guide/gems/reference/environment/landscape-canvas) to place and connect nodes in a visual graph that generates terrain. The advantage of using Landscape Canvas is that it creates and configures entities in the level based on your node graph, and it also provides a useful visual representation of the relationships between the various entities and components that generate the terrain.
+
+It's helpful to understand how the various gradient and terrain components work together, so it's recommended that you work through both versions of the tutorial in the tabs below.
+
+{{< tabs name="terrain-setup-tutorials" >}}
+
+{{% tab name="Terrain setup with entities" %}}
+
+In this section, you'll create a terrain spawner entity and a gradient entity. The terrain spawner entity defines the terrain and references a gradient that provides elevation data. The gradient entity provides the gradient (a heightmap image in this example) that contains the elevation data.
+
+### Create a terrain spawner entity
+
+1. Create a new entity (hotkey **Ctrl+Alt+N**). Name the entity `Terrain Spawner`.
+
+2. With the entity selected, in Entity Inspector, add a **Terrain Layer Spawner** component. This component enables the entity to generate terrain.
+
+3. The **Terrain Layer Spawner** component displays a warning about missing a required component. Choose **Add Required Component** and select **Axis Aligned Box Shape** from the list. This box shape defines the extents of the generated terrain.
+
+4. To generate terrain that is the same resolution as the heightmap, in the **Axis Aligned Box Shape** component, set the value of the **Dimensions:** **X** and **Y** properties to the resolution (in pixels) of the heightmap. The **Dimensions: Z** property can be used to scale the elevation. Enter a value for the **Dimensions: Z** property that roughly represents the maximum desired elevation in meters.
+
+    {{< tip >}}
+If the dimensions of the **Axis Aligned Box Shape** component are smaller (in world units) than the heightmap (in pixels), the resulting terrain has a lower fidelity might have sharp changes in elevation. You'll get the best results from a 1:1 mapping of heightmap pixels to terrain world units.
```

Comment:
I broke up a sentence and moved some words around at the end.

If the dimensions of the **Axis Aligned Box Shape** component are smaller (in world units) than the heightmap (in pixels), the resulting terrain will have a lower fidelity than the input heightmap. This might result in missing elevation data that causes unexpected smooth areas or areas of sharp change. You'll get the best results from a 1:1 mapping of heightmap pixels to terrain world units. This means that when the **Height Query Resolution** is set to 1 meter (the default), the box size in meters should be the same as the heightmap pixel resolution.

Reply:
Ooh - in re-reading, I thought of a different phrasing that's more specific on how to set the dimensions. What do you think of this?

For best results, the dimensions of the **Axis Aligned Box Shape** should be set to the heightmap pixel size multiplied by the **Height Query Resolution**. This ensures that there is exactly one heightmap pixel for each world unit used by the terrain system. When the Height Query Resolution is set to 1 meter (the default), the box size in meters should be exactly the same as the heightmap pixel resolution. If the box dimensions are too small, some heightmap pixels won't get used, which can cause unexpected smooth areas or areas of excessively sharp elevation changes.

Reply:
done